README.md

```diff
@@ -21,16 +21,16 @@
 4. If you are using Firefox, type `allow pasting`.
 5. Enter the following script:
 
-    ```js
-    const theme = [
-        '21=302D41', '7=1E1E2E', '8=D9E0EE', '9=F5E0DC', 'aa=C9CBFF',
-        'ae=1E1E2E', 'j=302D41', 'x=F5E0DC',
-    ];
+	```js
+	const theme = [
+		'21=302D41', '7=1E1E2E', '8=D9E0EE', '9=F5E0DC', 'aa=C9CBFF',
```

Comment:
Seems that GitHub did not pick up the indentation size from `.editorconfig` for use in `README.md`. Should the code blocks (or the whole file) be indented with spaces?